Composer/packages/client/src/recoilModel/DispatcherWrapper.tsx

```diff
@@ -72,7 +72,9 @@ const InitDispatcher = ({ onLoad }) => {
 
   const [currentDispatcherState, setDispatcher] = useRecoilState(dispatcherState);
 
-  useEffect(() => {
+  //The render order is different with 0.0.10, the local state will trigger a render before atom value
+  //so use the useLayoutEffect here
+  useLayoutEffect(() => {
```

Comment:
Do you mind explaining this change?

Reply:
Hi, @hatpick , it looks like that in 0.0.13, recoil removes the queue update in setRecoilState. So the recoil root state's update doesn't happen immediately. 
https://codesandbox.io/s/inspiring-dew-4r5xi?file=/src/App.js
this is the demo, you can change the recoil version to 0.0.10 and see the log message.
maybe this is updated by the performance https://recoiljs.org/blog/#performance

Reply:
@lei9444 I thought reason you added the change was inside InitDispatcher component the onLoad was fired before the dispatcher state is set using the setDispatcher fn. Essentially, we are left with a state where we load the children(entire app) before dispatcher is initialized. By moving it to the useLayout effect we synchronously set the values before rendering any children. Is that fair explanation?

Reply:
@srinaath That's matches what I understood.

Reply:
Thanks, @srinaath, Yes,  I just want to  sync the set function to make sure the children will get the dispatchers